install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -44,20 +58,27 @@ initContainers:
 {{- end }}
   securityContext:
     allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
+    privileged: {{ .Values.global.proxy.privileged }}
     capabilities:
+{{- if not $cniRepairEnabled }}
       add:
       - NET_ADMIN
       - NET_RAW
+{{- end }}
       drop:
       - ALL
-    privileged: {{ .Values.global.proxy.privileged }}
     readOnlyRootFilesystem: false
+  {{- if not $cniRepairEnabled }}
     runAsGroup: 0
     runAsNonRoot: false
     runAsUser: 0
+  {{- else }}
+    runAsGroup: 1337
```

Comment:
this does not seem right, why are we changing the permissions for cni repair disabled? That is probably why the tests are breaking

Reply:
It's a double-negative condition. This just sets the group to 1337 when running in CNI.